ci/conda-env.yml

```diff
@@ -54,3 +54,5 @@ dependencies:
 - python-utils
 - progressbar2
 - zipp<3.16.0
+- pip:
+  - lightgbm>=4.0.0
```

Comment:
re: flaky micromamba

we could replace setup-micromamba with setup-python:

you often see packages go `pip install pandas[dev]`. we could move this stuff into vaex['dev']` and deprecate this file? and we just go `pip install -e .[dev]` in github actions?

on a side-note: shouldn't the `lightgbm>=4.0.0` constraint go into vaex-ml pyproject.toml anyway?

Reply:
This file was there to quickly and reliably create a conda environment. This previously was impossible to do with pip.
I'm afraid that trying this will take a lot of time. Maybe we can merge this, do a release, and try to do this separately in a new PR?